sass/lightbox.scss

```diff
@@ -78,7 +79,6 @@ html.lightbox-enabled {
   .lightbox-contents {
     @extend %user-select-none;
 
-    cursor: zoom-out;
```

Comment:
As a side note, `cursor: "zoom-out"` does not work on IE11, so I had to change it to "pointer" for that  browser.